app/jobs/tracking_sheet_report_job.rb

```diff
@@ -0,0 +1,145 @@
+# frozen_string_literal: true
+
+##
+# Job to create downloadable tracking sheets
+# rubocop:disable Metrics/ClassLength
+class TrackingSheetReportJob < GenericJob
+  ##
+  # A job that...
+  # @param [Integer] bulk_action_id GlobalID for a BulkAction object
+  # @param [hash] params additional parameters that an Argo job may need
+  # @option params [Array] :druids required list of druids
+  def perform(bulk_action_id, params)
+    super
+
+    druids = params[:druids]
+
+    with_bulk_action_log do |log|
+      update_druid_count
+      begin
+        pdf = Prawn::Document.new(page_size: [5.5.in, 8.5.in])
+        pdf.font('Courier')
+        druids.each_with_index do |druid, i|
+          find_or_create_in_solr_by_id(druid)
```

Comment:
why do we do this here if we do it again in generate_tracking_sheet?

Reply:
I was refactoring to remove some duplication and missed this. Will fix.